src/Order.js

```diff
@@ -0,0 +1,20 @@
+import React ,{useState} from "react";
+import RestaurantButton from "./RestaurantButton";
+
+
+const Order = ({orderType})=>{
+
+const [orders,setOrders]=useState(0);
```

Comment:
you are storing an number, would it be better to define the variable as [order, setOrder]? giving it a plural implies it is an array.

Reply:
yes , you are rights , i was thinking it will lots of orders and that's why i wrote it like that , but i understand what you mean